docs/man/ctags.1.rst

```diff
@@ -1342,7 +1415,7 @@ Listing Options
 
 	ENABLED
 		Whether the kind is enabled or not. It takes ``yes`` or ``no``.
-		(Currently all roles are enabled. No option for disabling
+		(Currently all roles are enabled. An option for disabling
 		a specified role is not implemented yet.)
```

Comment:
I remember  I implemented an option for enabling/disabling roles:
```
--roles-<LANG>.kind=[+|-]role
```

Reply:
I've removed the note of the restriction.